static/docs/get-started/example-versioning.md

```diff
@@ -227,9 +227,10 @@ $ python train.py
 $ dvc add model.h5
 ```
 
-Note! `dvc remove` or `dvc unprotect` is required, otherwise `python train.py`
-will overwrite the existing file and may corrupt the cached version. Check this
-[guide](/doc/user-guide/update-tracked-file) to learn more.
+> **Note!** `dvc remove` or `dvc unprotect` is required, otherwise `python
```

Comment:
it's not required anymore. It's required only in the protected mode. By default it's disabled and it's safe to modify/overwrite files.

Reply:
Good point. Note removed.